### DISCUSSION TITLE
NPM package is not downloading

### DISCUSSION THREAD

**Discussion by nikitababko** [community/community#26538] [Actions]
<p>Such a problem:</p>
<p><strong>Launched a test project on NPM</strong> - <a href="https://www.npmjs.com/package/@nikitababko/test-repo-2" class="inline-onebox" rel="noopener nofollow ugc">@nikitababko/test-repo-2 - npm</a></p>
<p>Now I’m trying to download it: yarn add <a class="mention" href="/nikitababko">@nikitababko</a>/test-repo-2 <strong>or</strong> npm i <a class="mention" href="/nikitababko">@nikitababko</a>/test-repo-2.<br>
<strong>And throws an error</strong> - npm ERR! 401 Unauthorized - GET <a href="https://npm.pkg.github.com/@nikitababko%2ftest-repo-2" rel="noopener nofollow ugc">https://npm.pkg.github.com/@nikitababko%2Ftest-repo-2</a> - Your request could not be authenticated by the GitHub Packages service. Please ensure your access token is valid and has the appropriate scopes configured.</p>
<p><strong>Used actions:</strong></p>
<pre><code class="lang-auto">name: '🚀 publish'

on:
  push:
    branches:
      - master

jobs:
  release:
    name: 🚀 publish
    runs-on: ubuntu-latest
    steps:
      - name: 📚 checkout
        uses: actions/checkout@v2.3.4
      - name: 🟢 node
        uses: actions/setup-node@v2.1.5
        with:
          node-version: 14
          registry-url: https://registry.npmjs.org
      - name: 🚀 publish
        run: npm publish --access public
        env:
          NODE_AUTH_TOKEN: ${{secrets.NPM_AUTH_TOKEN}}
</code></pre>

**Comment by jcansdale** [comment, marked as answer]
<p>Hi <a class="mention" href="/nikitababko">@nikitababko</a>,</p>
<aside class="quote no-group" data-username="nikitababko" data-post="1" data-topic="180392">
<div class="title">
<div class="quote-controls"></div>
 nikitababko:</div>
<blockquote>
<p><a href="https://npm.pkg.github.com" rel="noopener nofollow ugc">https://npm.pkg.github.com</a></p>
</blockquote>
</aside>
<p>I’m confused about where it’s picking up <code>npm.pkg.github.com</code> from. Is this specified in your <code>package.json</code> file?</p>
<p>For example, like this:</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#publishing-a-package-using-publishconfig-in-the-packagejson-file" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#publishing-a-package-using-publishconfig-in-the-packagejson-file" target="_blank" rel="noopener nofollow ugc">Working with the npm registry - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#publishing-a-package-using-publishconfig-in-the-packagejson-file" target="_blank" rel="noopener nofollow ugc">//docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>If it isn’t, I’m surprised it isn’t picking up <code>registry-url: https://registry.npmjs.org</code> from your workflow!</p>
